composer.json

```
@@ -3,12 +3,12 @@
     "description":  "REST Library for the Discord API",
     "type":         "library",
     "require":      {
-        "php":                        "^5.6|^7.0",
+        "php":                        "^5.6|^7.0|^8.0",
         "guzzlehttp/guzzle":          "^6.3",
         "guzzlehttp/guzzle-services": "^1.0",
         "monolog/monolog":            "^1.22|^2.0",
         "netresearch/jsonmapper":     "^1.4",
-        "symfony/options-resolver":   "^2.6|^3.0|^4.0|^5.0"
+        "symfony/options-resolver":   "^2.6|^3.0"
```

Not sure this is a good change

```suggestion
"symfony/options-resolver":   "^3.4|^4.4|^5.1"
```

---

I was getting weird errors from Composer with this line earlier - but this seems to work